docs/setup/native_setup.md

```diff
@@ -14,34 +16,50 @@ You are now ready for the actual **OperationCode Backend** code base.
 * The following commands will pull down the source code from your forked repo.
 * Make sure to replace `[YOUR-GITHUB-NAME]` with your GitHub name. (example: https://github.com/iserbit/operationcode_backend.git)
 
-#### Local Development Environment
-_Fork the repo first._
 ```bash
 git clone https://github.com/[YOUR-GITHUB-NAME]/operationcode_backend.git operationcode-upstream
 cd operationcode-upstream
 git remote add upstream https://github.com/OperationCode/operationcode_backend.git
 ```
 
-1. If [Bundler](https://bundler.io/) is not installed run `gem install bundler`.
+### Ruby Version Manager
+
+RVM is a command-line tool which allows you to easily install, manage, and work with multiple ruby environments.
+
+1. [Install RVM](https://rvm.io/).
+2. You may need to restart terminal after the installation for the changes to take effect.
+3. Install the ruby version specified in the [`.ruby-version`](/.ruby-version) file. You can check the version by running `ruby -v`.
+4. Ensure ruby gemset from [`.ruby-gemset`](/.ruby-gemset) is applied. Run `rvm gemset list`. If a different gemset is applied, just run ``dir=`pwd` && cd .. && cd $dir``.
+
+### Install Dependencies
+
+1. Install [Bundler](https://bundler.io/) `gem install bundler`.
 2. Run `bundle install` to install all of the gems in your [Gemfile](Gemfile).
 3. Download and install [PostgreSQL](https://www.postgresql.org/download/) if you don't have it already.
 
 #### Database Setup:
 
-
-1. Run `cp config/database.yml. config/native_setup_database.yml`.
-2. Run `bin/rake db:create`.
-3. Run `bin/rake db:migrate`.
-4. Run `bin/rake db:seed` --> **This is where I am getting errors** :sweat_smile:.
+1. Run `bin/rails db:create`.
+2. Run `bin/rails db:migrate`.
+3. Run `bin/rails db:seed`.
 
 ### Testing
-1. Run `RAILS_ENV=test rake db:test:prepare`.
-2. Run `bin/rake test & rubocop`.
+
+1. Run `RAILS_ENV=test bin/rails db:test:prepare`.
+2. Run `bin/rails test`
```

Comment:
Nit picky but the other ones end with periods and this doesn't... personally I'd just drop all the periods since they make it harder to quickly copy paste the commands but it would be nice to keep them all consistent one way or the other.

Reply:
Yeah, I agree. Will remove the periods. Thanks